nrf_modem/doc/flavour_features.rst

```diff
@@ -1,16 +1,16 @@
-.. _nrf_modem_flavour:
+.. _nrf_modem_variants:
 
-Flavours and features
+Variants and features
```

Comment:
As per my understanding, with flavours we mean the cellular and DECT NR+.
Renamed the title as variants as I found "flavours" a bit odd. another term could  be "technology" in this case?

Reply:
Basically yes, you could call them `variant` but `flavour` is not that uncommon. It is used to distinguish between different versions of something, for example `vanilla` is a common flavour, meaning that it is the default or "plain" version.

I think it suits well in our case, mostly because we don't overload `variant`.

Among all these libraries, there are:
1.  differences in supported API
2. differences in ABI (hard-float, soft-float)
3. differences wrt logging (with or without)
4. differences in targeted SOC

Mainly I would like to distinguish between 1) and the rest, because 1) means that the library has different features, whereas 2), 3), 4) mean that the library has the same set of features offered in a different "package" let's say, for example, a different ABI or a different SOC.

So I have tried to use `flavour` for different APIs, and `variant` for all the rest.

This lets us say for example, that "the DECT PHY flavour has one variant for nRF9161 but not for nRF9160".
If we called everything variant, we could not say that "the DECT PHY variant has one variant for nRF9161 but not nRF9160".

Reply:
What's wrong with `flavour` then? It is used in computing to express just that.
Here's an example from Android docs: https://developer.android.com/build/build-variants, where they talk about "product flavors"

Reply:
Agreed offline we will use `variant`